### ISSUE TITLE
Question about RSA sigVer e length

### ISSUE THREAD

**Issue by jbarnesweb** [usnistgov/ACVP#1116]
This is the capability file I'm using to download a test session.

The question is about the pubExpMode attribute. When I set it to random, the e values I receive contain some 8-byte values, which are incompatible with a 32-bit library. FIPS 186-4 clearly clarifies the range of e as follows.

```
The exponent e shall be an odd positive integer such that:
2^16 < e < 2^56.
Note that the value of e may be any value that meets constraint 1(b), i.e., e may be
either a fixed value or a random value.
```

Is there a way to claim random capability for pubExpMode and constrain the test case e values to 32 bits?

```
[
	{
		"acvVersion": "1.0"
	},
	{
		"isSample": false,
		"algorithms": [
			{
				"algorithm": "RSA",
				"mode": "sigVer",
				"revision": "FIPS186-4",
				"prereqVals": [
					{
						"algorithm": "DRBG",
						"valValue": "same"
					},
					{
						"algorithm": "SHA",
						"valValue": "same"
					}
				],
				"pubExpMode": "random",
				"conformances": [
				],
				"capabilities": [
					{
						"sigType": "pkcs1v1.5",
						"properties": [
							{
								"modulo": 1024,
								"hashPair": [
									{
										"hashAlg": "SHA-1"
									},
									{
										"hashAlg": "SHA2-256"
									},
									{
										"hashAlg": "SHA2-384"
									},
									{
										"hashAlg": "SHA2-512"
									}
								]
							},
							{
								"modulo": 2048,
								"hashPair": [
									{
										"hashAlg": "SHA-1"
									},
									{
										"hashAlg": "SHA2-256"
									},
									{
										"hashAlg": "SHA2-384"
									},
									{
										"hashAlg": "SHA2-512"
									}
								]
							},
							{
								"modulo": 3072,
								"hashPair": [
									{
										"hashAlg": "SHA-1"
									},
									{
										"hashAlg": "SHA2-256"
									},
									{
										"hashAlg": "SHA2-384"
									},
									{
										"hashAlg": "SHA2-512"
									}
								]
							}
						]
					}
				]
			}
		]
	}
]

```

**Comment by celic** [comment]
There is not at the moment. This would require an update on the server to create the new registration property and tie it in.

**Comment by celic** [comment]
Duplicate #1061

**Comment by jbrock24** [comment]
Was able to track down the culprit behind this, the fix will go out with the next release. I'll post here when it's ready!

**Comment by livebe01** [comment]
The fix for this is on Prod in release [v1.1.0.31](https://github.com/usnistgov/ACVP-Server/releases/tag/v1.1.0.31).
